frontend/src/views/WheresMyGeneV2/components/GeneSearchBar/components/CitationButton/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { Notification } from "@czi-sds/components";
+import {
+  StyledNotificationWrapper,
+  StyledNotificationLabel,
+  StyledIcon,
+  StyledNotificationDetails,
+  StyledButtonDiv,
+  StyledLabel,
+  StyledButtonIcon,
+} from "./style";
+import { noop } from "src/common/constants/utils";
+import { EXCLUDE_IN_SCREENSHOT_CLASS_NAME } from "../SaveExport";
+import { useConnect } from "./connect";
+import {
+  CITATION_BUTTON_LABEL,
+  CITATION_NOTIFICATION_LABEL,
+  CITATION_NOTIFICATION_TEXT,
+} from "src/views/WheresMyGeneV2/common/constants";
+
+export default function CitationButton(): JSX.Element {
+  const { copyCitation, showCitationNotification, selectedGenes } =
+    useConnect();
+
+  return (
+    <>
+      {showCitationNotification > 0 && (
+        <StyledNotificationWrapper>
+          <Notification
+            key={showCitationNotification}
+            autoDismiss={5000}
+            onClose={noop}
+            slideDirection="left"
+            intent="info"
+            icon={
+              <StyledIcon sdsIcon={"copy"} sdsSize={"s"} sdsType={"static"} />
```

Comment:
Not sure if Figma is outdated, but I think it uses "quote" icon here too!

<img width="522" alt="Screenshot 2023-09-28 at 7 53 01 AM" src="https://github.com/chanzuckerberg/single-cell-data-portal/assets/6309723/747e9b3b-32cc-460e-a080-4461ec0f9a8a">

Reply:
we don't have it in this size - we only have the large one. I can ask Harley to iterate on this

Reply:
Ahh got it! So this is what Connor was saying about having an svg for small size in [Slack](https://czi-sci.slack.com/archives/C032S43KKFV/p1695839599949149?thread_ts=1695767006.796439&cid=C032S43KKFV) 

@hthomas-czi PTAL! In the meantime I think the copy icon looks good?

Thanks both!